presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/SemiTransactionalHiveMetastore.java

```diff
@@ -438,6 +439,50 @@ public synchronized void dropColumn(String databaseName, String tableName, Strin
         setExclusive((delegate, hdfsEnvironment) -> delegate.dropColumn(databaseName, tableName, columnName));
     }
 
+    public synchronized void updateTableParameters(ConnectorSession session, String databaseName, String tableName, Map<String, String> parametersToUpdate, Set<String> parametersToDrop)
```

Comment:
We should make it a generic interface:
```java
alterTable(ConnectorSession session, String databaseName, String tableName, Table newTable)
```

Then, in this function, we should verify everything else is the same and update the table params only. If we found something else is different (e.g., owner, column, etc), we should fail it. Same, even for table params, we should only touch `PRESTO_DEPENDENT_MATERIALIZED_VIEW_LIST`. 

Add a javadoc to indicate the functionality is very limited